website/public/locale/en/translation.yaml

```diff
@@ -89,7 +94,7 @@ newsletter:
 joinCommunity:
   title: Join our friendly community
   slackAlt: Join on Slack
-  slackTitle: 'Get invitation to join #OpenEBS on Kubernetes Slack'
+  slackTitle: "Get invitation to join #OpenEBS on Kubernetes Slack"
```

Comment:
Why quote for this one particular string ?

Reply:
These changes are not done now, it changed when the document was formatted. Though, the reason for quotes for this string was it has a special character `#` in order to handle that quotes were required.

Reply:
cc @Pallavi-PH

Reply:
Yes, in order to include special characters as string in translation we need to wrap in in quotes